Features.md

```diff
@@ -0,0 +1,24 @@
+## JGit
+### Feature 1
+  #### Story A - Add Pull Command Line Command
+   - Task 1: Add a pull command to the CLI, callable with "jgit pull"
+   - Task 2: Add remote and remote branch arguments to pull command
+   - Task 3: Add a test option to pull command, which outputs raw data about pull (fetch and merge outputs)
+   - Task 4: Add rebase options to pull command 
```

Comment:
Does it make sense to provide a link to Git documentation that defines terms like "rebase".

Reply:
Yes it does. I will add them